src/frontend/typer/person.ts

```diff
@@ -60,6 +60,7 @@ export interface ISøker extends Omit<ISøkerRespons, 'barn'> {
     harSamboerNå: ISøknadSpørsmål<ESvar | null>;
     nåværendeSamboer: ISamboer | null;
     utenlandsperioder: IUtenlandsperiode[];
+    arbeidsperioder: IArbeidsperiode[];
```

Comment:
Kanskje vi kan droppe å sette noe på søknadsobjektet denne runden, og lage en egen PR for de 3 modalene vi har nå? Litt usikker på hvordan det vil bli med mellomlagring ogsånn

Reply:
Fjerner denne også